### PR TITLE
Created functionality to change tasks status 💪

### PR DESCRIPTION
**Pull request by hsztan**
I was able to create + an event handler to listen 🦻 for a mouse click 🐁 event to style 🖌 the selected task item 📝 and toggle 🔄 its completion status; as well as persisting 💾 the change in the browser's local storage 🤓

> At the moment, Github is having trouble running linter workflows. I ran them locally with no problems.

### CODE REVIEW

File: src/modules/startApp.js [hunk 1]
```diff
@@ -128,22 +137,68 @@ const createTaskElem = (task) => {
...
   });
+  // resetTasksIndexes();
+};
+
+const removeAllTasksFromDom = () => {
+  todoContainerElem.innerHTML = '';
 };
 
+// create event listener to clear all completed tasks
+const clearAllCompletedTasks = () => {
```
Comment (t-yanick) [inline src/modules/startApp.js:190]: - [ ] The function(`button`) to clear all complemented tasks is not working, please kindly fix the issue as it is a requirement for this project.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @hsztan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/startApp.js (hunk 1 above)
Comment (hsztan) [reply src/modules/startApp.js:190]: @t-yanick Right! 😳 I put the functionality in the little reload icon on the top right corner. I will also add it to the main button 👍

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @hsztan,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**